### PR TITLE
Install icon properly

### PR DESCRIPTION
**Pull request by albfan**
Solve #1

### CODE REVIEW

File: com.vesc_project.VescTool.yaml [hunk 1]
```diff
@@ -22,7 +22,7 @@ modules:
   - name: metainfo
     buildsystem: simple
     build-commands:
-      - install -D -m0644 com.vesc_project.VescTool.png /app/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png
+      - install -D -m0644 com.vesc_project.VescTool.png /app/share/icons/flatpak/128x128/com.vesc_project.VescTool.png
```
Comment (bilelmoussaoui) [inline com.vesc_project.VescTool.yaml]:
```suggestion
      - install -D -m0644 com.vesc_project.VescTool.png /app/share/icons/hicolor/128x128/com.vesc_project.VescTool.png
```

Comment (albfan) [reply com.vesc_project.VescTool.yaml]: ouch

**Comment by albfan** [comment]
@bilelmoussaoui doesn't seems to work. 

```
zgrep "<icon type=\"remote\">" builddir/*/share/app-info/xmls/com.vesc_project.VescTool.xml.gz || test -f builddir/*/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png
 in dir /srv/buildbot/worker/build-x86_64-7/build (timeout 1200 secs)
 watching logfiles {}
 argv: b'zgrep "<icon type=\\"remote\\">" builddir/*/share/app-info/xmls/com.vesc_project.VescTool.xml.gz || test -f builddir/*/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png'
 using PTY: False
program finished with exit code 1
elapsedTime=0.014104
```

File: com.vesc_project.VescTool.yaml [hunk 2]
```diff
@@ -22,7 +22,7 @@ modules:
   - name: metainfo
     buildsystem: simple
     build-commands:
-      - install -D -m0644 com.vesc_project.VescTool.png /app/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png
+      - install -D -m0644 com.vesc_project.VescTool.png /app/share/icons/128x128/com.vesc_project.VescTool.png
```
Comment (bilelmoussaoui) [inline com.vesc_project.VescTool.yaml]:
```suggestion
      - install -D -m0644 com.vesc_project.VescTool.png /app/share/icons/hicolor/128x128/apps/com.vesc_project.VescTool.png
```

Comment (albfan) [reply com.vesc_project.VescTool.yaml]: thanks!

**Comment by albfan** [comment]
@bilelmoussaoui I just added icon two times.

Currently master shows the icon

Captura de pantalla de 2021-02-18 15-18-14

we are just trying to make flathub to show the icon too.

File: com.vesc_project.VescTool.yaml [hunk 3]
```diff
@@ -23,6 +23,7 @@ modules:
     buildsystem: simple
     build-commands:
       - install -D -m0644 com.vesc_project.VescTool.png /app/share/app-info/icons/flatpak/128x128/com.vesc_project.VescTool.png
```
Comment (hfiguiere) [inline com.vesc_project.VescTool.yaml]: I don't think you should install anything in there.

Comment (albfan) [reply com.vesc_project.VescTool.yaml]: How should it be done?

**Comment by hfiguiere** [comment]
build fail because
```
builddir/files/share/appdata/com.vesc_project.VescTool.appdata.xml: FAILED:
• attribute-invalid     : <screenshot> width (1077) did not match specified (1276) [https://vesc-project.com/sites/default/files/vesc_tool.png]
• attribute-invalid     : <screenshot> height (609) did not match specified (768) [https://vesc-project.com/sites/default/files/vesc_tool.png]
Validation of files failed
```

**Comment by AsciiWolf** [comment]
Any chance this PR could be merged along with other updates? It looks like that this Flatpak sadly got abandoned.

**Comment by hfiguiere** [comment]
it needs work since the build failed.

**Comment by hfiguiere** [comment]
new error 

```
error: /srv/buildbot/worker/build-x86_64-6/build/builddir/export/share/icons/hicolor/128x128/apps/com.vesc_project.VescTool.png is not a valid icon: Expected a square icon but got: 157x139
```

**Pull request closed without merging**